libvast/include/vast/transform_steps/select.hpp

```diff
@@ -18,22 +18,26 @@ struct select_step_configuration {
   // The expression in the config file.
   std::string expression;
 
+  // Whether to select or to filter.
+  bool invert = false;
```

Comment:
We have both `project` and `delete` steps, so why not introduce a `filter` step whose plugin implementation simply refers to the `filter` plugin?